appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/processor/ApplicationProcessor.java

```diff
@@ -457,24 +438,9 @@ private void addParameter(AnnotatedElement element, ApiContext context,
         if (context.getWorkingOperation() != null) {
             context.getWorkingOperation().addParameter(newParameter);
         } else {
-            if (element instanceof Field) {
-                Field field = Field.class.cast(element);
-                ApiContext apiContext;
```

Comment:
FYI: Below logic moved to the walker so the walker now is the only one creating the context which is both cleaner and was needed to allow moving inner class processing to the walker what needed to delay generation of resource mapping which also moved to the walker.